pkgs/by-name/ta/tangle/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeDesktopItem,
+  fetchurl,
+  # Build depends
+  cmake,
+  qt6,
+  zlib,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tanglet";
+  version = "1.6.5";
+
+  src = fetchFromGitHub {
+    owner = "gottcode";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-xhFxH6yBKZwI5+5BX5F4H325h2V7hOLe7UocUMcrQAo=";
+  };
+
+  buildInputs = [
+    cmake
+    qt6.full
+    zlib
+  ];
+
+  desktopItem = makeDesktopItem {
+    name = pname;
+    desktopName = "Tanglet";
+    comment = "Single player word finding game based on Boggle";
+    exec = "tanglet";
+    icon = fetchurl {
+      url = "https://github.com/gottcode/tanglet/raw/v${version}/icons/hicolor/64x64/apps/tanglet.png";
+      hash = "sha256-K3mL/GXWKfBU0jV59P3/h6BIojICG85iltv4nrrlEq4=";
+    };
+    categories = [ "Game" ];
+  };
```

Comment:
This is not needed, tanglet is already doing it in `CMakeLists.txt` already does it.